ide-support/revliburl.livecodescript

```diff
@@ -2058,7 +2058,7 @@ function ulBuildHttpRequest pUrl
          end if
       end repeat
       if laSocketSecured[tSocket] is false then
-         put "CONNECT" &&  laUrlFormat[pUrl]["urlHost"] & ":" & laUrlFormat[pUrl]["urlPort"] && "HTTP/1.1" & cr & "Host:" && laUrlFormat[pUrl]["urlHost"] & ":" & laUrlFormat[pUrl]["urlPort"] into tRequest
+         put "CONNECT" &&  laUrlFormat[pUrl]["urlHost"] & laUrlFormat[pUrl]["urlPort"] && "HTTP/1.1" & cr & "Host:" && laUrlFormat[pUrl]["urlHost"] & laUrlFormat[pUrl]["urlPort"] into tRequest
```

Comment:
@trevordevore is the colon meant to be stripped from urlHost or urlPort and it isn't being done? Just a bit concerned the bug is elsewhere.